### ISSUE TITLE
Add 2.1.x version alias for Django docs

### ISSUE THREAD

**Issue by timgraham** [rtfd/readthedocs.org#4519]
## Details
- Project URL: https://readthedocs.org/projects/django/
## Request
As in https://github.com/rtfd/readthedocs.org/issues/2285, please add version 2.1.x as an alias for the stable/2.1.x branch just like have for other branches.

**Comment by stsewd** [comment]
This would be soon possible from the admin panel https://github.com/rtfd/readthedocs.org/pull/4505, btw, aren't the django docs hosted in https://docs.djangoproject.com/?

**Comment by timgraham** [comment]
Yes, we use readthedocs as a backup and for PDFs/offline versions downloads.

**Comment by humitos** [comment]
Hi @timgraham! As @stsewd mentioned this will be possible soon. Please, follow #4505 to know when it's merged and our changelog (http://docs.readthedocs.io/en/latest/changelog.html) to know when it's deployed.

> NOTE: `rel` branch is the one deployed in production

We are in a development-freeze week because we are migrating servers (https://blog.readthedocs.com/azure-migration/). So, I estimate that this feature will be available in 2 weeks.

If that's not enough for you, let me know and I could tweak/hack the database as Eric/Anthony has done before and add this particular version to be built.

**Comment by timgraham** [comment]
It would be nice to have this addressed asap since Django 2.1 has been released since August 1 and we have some broken links on our site without this.

**Comment by humitos** [comment]
@timgraham I just manually created the version `2.1.x` for you and triggered a build of this version. It's building at https://readthedocs.org/projects/django/builds/7640924/

You can see this version at https://django.readthedocs.io/en/2.1.x/ when the build finishes.

Let me know if you find something weird or broken after this. Hopefully I didn't break anything.

**Comment by humitos** [comment]
Glad to hear that! :smiley:
